helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -5,6 +5,11 @@ The values can be overridden in .Values.clusterReceiver.config
 {{- define "splunk-otel-collector.clusterReceiverConfig" -}}
 {{ $clusterReceiver := fromYaml (include "splunk-otel-collector.clusterReceiver" .) -}}
 extensions:
+  {{- if (eq (include "splunk-otel-collector.persistentQueueEnabled" .) "true") }}
+  {{- include "splunk-otel-collector.persistentQueueLogs" (dict "Values" .Values "forAgent" false) | nindent 2 }}
+  {{- include "splunk-otel-collector.persistentQueueMetrics" (dict "Values" .Values "forAgent" false) | nindent 2 }}
```

Comment:
you are defining both file storage extensions regardless of whether only one is set, and therefore might get errors because the folder is not created.

Reply:
yeah, will resolve this.